Felienne/hedy
# [NEW FEATURE] Classes overview in admin page

**Description**
In this PR we add a classes overview to the admin page. As the admin page was already pretty crowded, a restructuring is done as well. The `/admin` page is an overview with the option to select one of the following subsections:

- Users overview
- Classes overview
- Statistics

**Fix for**
This PR fixes #1741 and fixes #1783

**How to test**
Navigate to `/admin` (make sure you are admin in `os.getenv()`) and notice the new UI. Verify that all buttons are correctly working to direct the page to the selected button.

**Screenshots**
![Schermafbeelding 2022-01-25 om 12 06 30](https://user-images.githubusercontent.com/38155424/150965836-932da623-d997-42d5-84ba-d2af6d4803ca.png)
![Schermafbeelding 2022-01-25 om 12 06 40](https://user-images.githubusercontent.com/38155424/150965858-2620316f-387d-45df-884a-808f03b21b0d.png)
